home.html

```diff
@@ -0,0 +1,165 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link
+    rel="stylesheet"
+    href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/4.1.1/animate.min.css"
+  />
+
+    <title>Capstone project</title>
+</head>
+<!-- social section  -->
+<header class="social">
+    <img src="./images/facebook (1).png" alt="facebook" class="facebook">
+    <img src="./images/twitter (1).png" alt="twitter" class="twitter">
+
+    <ul>
+        <li><a href="#">English</a></li>
+        <li><a href="#">My Page</a></li>
+        <li><a href="#">Logout</a></li>
+    </ul>
+</header>
+<body>
+
+    <!-- mobile navigation  -->
+<div class="head">
+    <nav class="navbar">
+        <div class="navv">
+            <div class="ham">
+                <ion-icon name="menu"></ion-icon>
+                <ion-icon name="close-outline" class="close-btn"></ion-icon>
+            </div>
+            <div class="logoo">
+                <div class="logoimg"><a href="./home.html"><img src="./images/thelogo.png" alt="logo" width="100px" height="90px"></a></div>
+                <div class="text">
+                    <p>Click logo!</p>
+                </div>
+            </div>
+        </div>
+        <div class="list_nav">
+            <ul>
+                <li><a href="./home.html">Home</a></li>
+                <li><a href="./about.html">About</a></li>
+                <li><a href="#project">Project</a></li>
+                <li><a href="#join">Join</a></li>
+                <li><a href="#sponsor">Sponsor</a></li>
+                <li><a href="#news">News</a></li>
+            </ul>
+            <button type="submit" class="campaign">Learn more</button>
+        </div>
```

Comment:
- [ ] [OPTIONAL] it would be great if you can add some background opacity with blur to the mobile menu, I think this will make the mobile menu more like the design